stable/ark/templates/config.yaml

```diff
@@ -0,0 +1,49 @@
+{{- $root := . }}
+{{ with .Values.configuration }}
```

Comment:
If you move this further down as it was, you don't need to capture the root context in a separate variable.